Source/WebGPU/WGSL/Parser.cpp

```diff
@@ -1102,6 +1114,26 @@ Result<AST::Expression::List> Parser<Lexer>::parseArgumentExpressionList()
     return { WTFMove(arguments) };
 }
 
+template<typename Lexer>
+Result<AST::Value::Ref> Parser<Lexer>::parseConstantValue()
```

Comment:
This looks identical to `parseLetValue`, except for the keyword and return type. Maybe they could share the implementation?

Reply:
I did consider doing the same but then err'd on the side of simplicity and having two functions. But I'm not precious either way.